source/extensions/tracers/xray/daemon.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+// The structures in this file are merely for JSON serialization to communicate with the X-Ray
+// daemon. The X-Ray daemon does not speak protobuf.
+package envoy.tracers.xray.daemon;
```

Comment:
@marcomagdy How come this is in the `envoy.tracers` namespace? This seems likely to cause some confusion with API protos which share the namespace. I hit this while developing some tooling around the APIs that noticed this type.

Reply:
Happy to change it. I wasn't super sure what namespace it should go under. I couldn't find many examples of proto files in non-api locations.

should it be `envoy.extensions.tracers.xray.daemon`?

Reply:
If it's just used in a non-API way, locally by this extension, I would name it by path, i.e. `source.extensions.tracers.xray.daemon`. See https://github.com/envoyproxy/envoy/blob/master/test/fuzz/common.proto for precedent. Thanks for looking into this.

Reply:
@htuch here it is - https://github.com/envoyproxy/envoy/pull/9350